src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPackageInstaller.cs

```diff
@@ -31,9 +31,9 @@ public class VsPackageInstaller : IVsPackageInstaller2
         private readonly ISourceRepositoryProvider _sourceRepositoryProvider;
         private readonly Configuration.ISettings _settings;
         private readonly IVsSolutionManager _solutionManager;
-        private readonly INuGetProjectContext _projectContext;
         private readonly IVsPackageInstallerServices _packageServices;
         private readonly IDeleteOnRestartManager _deleteOnRestartManager;
+        private INuGetProjectContext ProjectContext;
```

Comment:
As per [Coding Guidelines](https://github.com/NuGet/Home/wiki/Coding-Guidelines#coding-style-guidelines--general) `_camelCase` is a convention for any private member regardless of the readonly.